google/_async_resumable_media/__init__.py

```diff
@@ -52,10 +52,10 @@
 
 
 __all__ = [
-    u"DataCorruption",
-    u"InvalidResponse",
-    u"PERMANENT_REDIRECT",
-    u"RetryStrategy",
-    u"TOO_MANY_REQUESTS",
-    u"UPLOAD_CHUNK_SIZE",
+    "DataCorruption",
+    "InvalidResponse",
+    "PERMANENT_REDIRECT",
+    "RetryStrategy",
+    "TOO_MANY_REQUESTS",
+    "UPLOAD_CHUNK_SIZE",
```

Comment:
There are also quite a few occurrences of single quoted strings with this `u` prefix.

Also, the script replaced `u` bytes in binary files, i.e. test images, which is probably why a few system tests failed.

Reply:
- `u'` literals: e7cdd35
- Images restored: 9d82941
- Systest import: e2b99bd